ci-operator/step-registry/telco5g/jobs-router/telco5g-jobs-router-commands.sh

```diff
@@ -10,45 +10,81 @@ echo "************ telco cluster setup command ************"
-echo "export DEBUG_PROW_JOB_ID=${PROW_JOB_ID}" >> $MAINENV
-echo "export DEBUG_JOB_NAME=${JOB_NAME}" >> $MAINENV
-echo "export DEBUG_JOB_TYPE=${JOB_TYPE}" >> $MAINENV
-echo "export DEBUG_RELEASE_IMAGE_LATEST=${RELEASE_IMAGE_LATEST-}" >> $MAINENV
-echo "export GIT_COMMITTER_NAME='CI User'" >> $MAINENV
-echo "export GIT_COMMITTER_EMAIL='cnf-devel@redhat.com'" >> $MAINENV
-echo "#######################################################" >> $MAINENV
+cat << EOF > "$MAINENV"
+# THERE IS THE ENVS FILE FOR JOBS
+export DEBUG_T5CI_JOB_TYPE='${T5CI_JOB_TYPE}'
+export DEBUG_T5CI_VERSION='${T5CI_VERSION}'
+export DEBUG_PROW_JOB_ID='${PROW_JOB_ID}'
+export DEBUG_JOB_NAME='${JOB_NAME}'
+export DEBUG_JOB_TYPE='${JOB_TYPE}'
+export DEBUG_RELEASE_IMAGE_LATEST='${RELEASE_IMAGE_LATEST-}'
+export GIT_COMMITTER_NAME='CI User'
+export GIT_COMMITTER_EMAIL='cnf-devel@redhat.com'
+#######################################################
+EOF
 
 if [[ "$PROW_JOB_ID" = *"nightly"* ]] && [[ "$JOB_TYPE" == "periodic" ]]; then
-    echo "export T5_JOB_TRIGGER=nightly" >> $MAINENV
-    echo "export T5_JOB_DESC=nightly-${T5CI_JOB_TYPE}" >> $MAINENV
-    echo "export T5_JOB_DESC_FULL=nightly-${T5CI_JOB_TYPE}-${T5CI_VERSION}" >> $MAINENV
     if [[ -n "${RELEASE_IMAGE_LATEST-}" ]]; then
         export IMG=${RELEASE_IMAGE_LATEST-}
     elif [[ -n "${RELEASE_IMAGE_INITIAL-}" ]]; then
         export IMG=${RELEASE_IMAGE_INITIAL-}
     else
         export IMG=${PROW_JOB_ID/-telco5g/}
     fi
-    echo "export T5_JOB_IMAGE=${IMG}" >> $MAINENV
-    # In case of running on nightly releases we need to figure out what release exactly to use
-    echo "export KCLI_PARAM=\"-P openshift_image=registry.ci.openshift.org/ocp/release:${IMG}\"" >> $MAINENV
-    echo "export SNO_PARAM=\"-f ${IMG}\"" >> $MAINENV
+    IMG_URL="registry.ci.openshift.org/ocp/release:$IMG"
+
+    cat << ____EOF >> "$MAINENV"
+export T5_JOB_TRIGGER=nightly
+export T5_JOB_DESC='nightly-${T5CI_JOB_TYPE}'
+export T5_JOB_DESC_FULL='nightly-${T5CI_JOB_TYPE}-${T5CI_VERSION}'
+# In case of running on nightly releases we need to figure out what release exactly to use
+export T5_JOB_RELEASE_IMAGE='$IMG_URL'
+export KCLI_PARAM='-P openshift_image=$IMG_URL'
+export SNO_PARAM='-f ${IMG}'
+____EOF
+
 elif [[ "$JOB_TYPE" == "periodic" ]]; then
-    echo "export T5_JOB_TRIGGER=periodic" >> $MAINENV
-    echo "export T5_JOB_DESC=periodic-${T5CI_JOB_TYPE}" >> $MAINENV
-    echo "export T5_JOB_DESC_FULL=periodic-${T5CI_JOB_TYPE}-${T5CI_VERSION}" >> $MAINENV
-    echo "export KCLI_PARAM=\"-P tag=${T5CI_VERSION} -P version=nightly\"" >> $MAINENV
-    echo "export SNO_PARAM=\"-t ${T5CI_VERSION} -r nightly\"" >> $MAINENV
+    # Identify release image url
+    # Ref.: https://github.com/karmab/kcli/blob/main/kvirt/cluster/openshift/disconnected/scripts/03_mirror.sh
+    if [ "$(echo "$T5CI_VERSION" | tr -dc '.' | wc -c)" -ge 2 ]; then
+        TAG="$T5CI_VERSION"
+    else
+        TAG="latest-$T5CI_VERSION"
+    fi
```

Comment:
Not sure we need all this check, we always run on `latest-$T5CI_VERSION`. `T5CI_VERSION` is set by us in the job definiiton

Reply:
Dropped, done.